modules/assim.sequential/R/aggregate.R

```diff
@@ -0,0 +1,36 @@
+##' @title Aggregation Function
+##' @name aggregate
+##' @author Harunobu Ishii
+##'
+##' @param downscale_output  Raster file output from downscale_function.R. Read file in this way if stored locally: \code{downscale_output <- readRDS("xxx.rds")}
+##' @param polygon_data A spatial polygon object (e.g., an `sf` object) that defines the spatial units for aggregation. 
+##'                     This data should be in a coordinate reference system compatible with the raster data (e.g., "EPSG:4326").
+##'                     Example of state-level aggregation: 
+##'                     \code{
+##'                     us_states <- readRDS("polygon/us_states.rds")
+##'                     state <- "MA"
+##'                     polygon_data <- st_transform(us_states[us_states$STUSPS == state, ], crs = "EPSG:4326")
+##'                     }
+##' @details This function will aggregate previously downscaled carbon flux amount to a spatial unit of choice 
+##'
+##' @return It returns the `polygon_data` with added columns for mean and sum values of the aggregated raster data for each ensemble member.
```

Comment:
you'll need to run `make document` to build the man file for this function and then add the man file to the PR